lago/prefix.py

```diff
@@ -805,6 +805,7 @@ def virt_conf_from_stream(
         Returns:
             None
         """
+        virt_conf = collections.OrderedDict()
```

Comment:
This is effectively doing nothing, as it's overriden on the next line (with whatever load_virt_stream returns)